TradeItIosTicketSDK2/TradeItPortfolioAccountDetailsViewController.swift

```diff
@@ -24,6 +26,7 @@ class TradeItPortfolioAccountDetailsViewController: TradeItViewController, Trade
         self.tableViewManager.delegate = self
         self.tableViewManager.table = self.table
 
+        self.fetchBrokerSupportedServices()
```

Comment:
What if the user changes account to select a different broker?

Reply:
We hit again `viewDidLoad` and this method